tests/crosstest/kitex2dubbo/base_type_test.go

```diff
@@ -98,6 +98,46 @@ func TestMain(m *testing.M) {
 	os.Exit(0)
 }
 
+func TestEchoRetBool(t *testing.T) {
```

Comment:
These tests should end with _Java and invoked by cli2Java.

Reply:
addressed